manifest/schema1/manifest.go

```diff
@@ -49,59 +97,78 @@ type Manifest struct {
 }
 
 // SignedManifest provides an envelope for a signed image manifest, including
-// the format sensitive raw bytes. It contains fields to
+// the format sensitive raw bytes.
 type SignedManifest struct {
 	Manifest
 
-	// Raw is the byte representation of the ImageManifest, used for signature
-	// verification. The value of Raw must be used directly during
-	// serialization, or the signature check will fail. The manifest byte
+	// Canonical is the canonical byte representation of the ImageManifest,
+	// without any attached signatures. The manifest byte
 	// representation cannot change or it will have to be re-signed.
-	Raw []byte `json:"-"`
+	Canonical []byte `json:"-"`
+
+	// all contains the byte representation of the Manifest including signatures
+	// and is retuend by Payload()
+	all []byte
```

Comment:
`Payload()` is the only available function in the Manifest API to get access to the bytes.  This is used by the HTTP endpoint for retrieving a manifest (previously this was accessing .`Raw`) directly.  All other accesses to the manifest bytes are within the schema1 context and operate only on the actual manifest bytes (stored in `Canonical`)